src/components/bubble/_bubble.scss

```diff
@@ -4,9 +4,10 @@ $bubbleBorderColor: $graySecondary;
 $bubbleBorderRadius: 10px;
 $bubbleBoxShadow: $defaultBottomShadow;
 
-$borderSize: 3px;
-$triangleSize: 12px;
-$innerTriangleSize: $triangleSize - 2 * $borderSize + 1;
+$borderSize: rhythm(1/8);
+$triangleSize: rhythm(1/2);
+$innerTriangleSize: rhythm(0.366);     // hand-picked value, checked against basic font-size changes, scales 11px - 30px
```

Comment:
There should be some math underneath this values.
If there will be any changes to this triangle it may be a good idea to change it to formula.